server/server.js

```diff
@@ -148,6 +148,24 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.put("/admin/administrators", function(req, res) {
+        try {
+            googleAuthoriser.addAdmin(req.body.email);
+            res.sendStatus(200);
+        } catch (err) {
+            res.sendStatus(404);
```

Comment:
You can just pass an argument to the reject function, i.e. reject(500) to pass 500 to the catching catch statement